<!-- 
rosrun stage_ros stageros /home/rr/ROSwork/Assignment1/src/assignment1/world/exercise.world &
-->

<launch>
	<node pkg="stage_ros" name="stageros_simulator" type="stageros" args="$(find robot_game)/world/exercise.world" required="true"/>
	<node pkg="robot_game" name="rg_services_py" type="rg_services_py.py" output="screen" />
	<node pkg="robot_game" name="rg_controller_py" type="rg_controller_py.py" required="true" output="screen" />
</launch>
